README.md

```diff
@@ -1,14 +1,31 @@
 # Northcoders News API
+Project summary:
 
-For instructions on how to create environment variables for anyone who wishes to clone my project and run it locally.
-1. Create a file named .env.test in the root directory of the project for your test database.
-2. Inside of the .env.test file add PGDATABASE=nc_news_test.
-3. Create a file named .env.development in the root directory of the project for your dev database.
-4. Inside of the development file similarly as before add PGDATABASE=nc_news.
-5. The config is already required in the connection file inside db.
+The purpose of this project is to build an API to access application data programmatically. The intention is to mimic the building of a real-world backend service (such as Reddit) which should provide this information to the front-end architecture.
 
+To access the hosted version, please visit https://magdaleenas-nc-news.onrender.com/api
 
+---
 
+To clone the project, follow these steps:
+1. Click on 'Code' and copy the HTTPS link
+2. Select a folder in your local machine and run the command: git clone [paste HTTPS link here, remove brackets].
+
+Next, you will need to create the necessary environment variables to run it locally:
+1. Create a file named .env.test in the root directory of the project to configure your testing database.
+2. In the .env.test file, declare the variable PGDATABASE=nc_news_test.
+3. Similarly, create a file named .env.development in the root directory for your development database.
+4. In the development file, specify PGDATABASE=nc_news.
```

Comment:
It might be better to not to mention the actual database name here for safety reasons